README.md

```diff
@@ -11,11 +11,11 @@ Test data (ADaM) for the pharmaverse family of packages
 
 # Purpose
 
-To provide a one-stop-shop for ADaM test data in the pharmaverse family of packages.
+To provide a one-stop-shop to create ADaM test data in the pharmaverse family of packages.
```

Comment:
I don't think this is an improvement in the language. Reasoning: pharmaverseadam **already has** the test data, the expectation is that users just use the data from here.

Reply:
you are correct;   I restored original language.